R/fit_seir.R

```diff
@@ -324,12 +332,19 @@ fit_seir <- function(daily_cases,
     i0 <- stats::rlnorm(1, i0_prior[1], i0_prior[2]/2)
     start_decline <- stats::rlnorm(1, start_decline_prior[1], start_decline_prior[2]/2)
     end_decline <- stats::rlnorm(1, end_decline_prior[1], end_decline_prior[2]/2)
-    f <- stats::rbeta(
+    f <- stats::rbeta( # FIX?
       1,
-      get_beta_params(f_prior[1], f_prior[2]/4)$alpha,
-      get_beta_params(f_prior[1], f_prior[2]/4)$beta
+      get_beta_params(f_prior[1,1], f_prior[1,2]/4)$alpha,
+      get_beta_params(f_prior[1,1], f_prior[1,2]/4)$beta
     )
-    f_s <- array(f, dim = stan_data$S)
+    f_s <- array(0, dim = stan_data$S) 
+    for (s in 1:stan_data$S) {
+        f_s[s] <- stats::rbeta( # FIX?
+              1,
+              get_beta_params(f_prior[s,1], f_prior[s,2]/4)$alpha,
```

Comment:
Dividing by 4 was completely arbitrary, just FYI. It would be better if we started from a broader initialization, but sometimes the ODE solver would get stuck.